src/actions/ens/getEnsAddress.ts

```diff
@@ -68,9 +69,14 @@ export async function getEnsAddress<TChain extends Chain | undefined,>(
     blockNumber,
     blockTag,
   })
-  return decodeFunctionResult({
+
+  if (res[0] === '0x') return null
+
+  const address = decodeFunctionResult({
     abi: singleAddressResolverAbi,
     functionName: 'addr',
     data: res[0],
   })
+
+  return trim(address) === '0x' ? null : address
```

Comment:
```suggestion
  return trim(address) === '0x0' ? null : address
```

Can we add a test for this line?